### PR TITLE
Fix client dropdown, streamlining form styling, fix menu highlighting

### PR DESCRIPTION
**Pull request by jeppekroghitk**
#### Link to ticket

https://leantime.itkdev.dk/tickets/showKanban?search=true&projectId=37&sprint=all#/tickets/showTicket/613

#### Description

This PR serves to tie up diverse loose ends, including:
- Adding filters and sorting to Client and Account view.
- Fixing menu highlight post introduction of views.
- Streamlining some classes and styling regarding the filter row of many pages.
- Added display names when hiding rows in Planning overview.

#### Screenshot of the result

N/A

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

### CODE REVIEW

File: src/Controller/InvoiceController.php [hunk 1]
```diff
@@ -188,8 +184,8 @@ public function edit(Request $request, Invoice $invoice, InvoiceRepository $invo
 
         // Only allow adding entries when material number and receiver account have been set.
         $allowAddingEntries = !empty($invoice->getDefaultReceiverAccount())
-            && !empty($invoice->getDefaultMaterialNumber())
-            && !empty($invoice->getDefaultMaterialNumber()->value);
+          && !empty($invoice->getDefaultMaterialNumber())
```
Comment (tuj) [inline src/Controller/InvoiceController.php:187]: 4 space indent

File: src/Controller/ViewController.php [hunk 2]
```diff
@@ -49,7 +49,7 @@ public function __construct(
     }
 
     #[Route(['/', '/list'], name: 'app_view_list')]
-    public function list(ViewRepository $viewRepository): Response
+    public function list(Request $request, ViewRepository $viewRepository): Response
```
Comment (tuj) [inline src/Controller/ViewController.php:52]: Why? It is not used...

File: templates/planning/index.html.twig [hunk 3]
```diff
@@ -5,12 +5,12 @@
 {% block content %}
     <h1 class="page-title">{{ 'planning.title'|trans }}</h1>
 
-    <div class="selections">
         {{ form_start(form) }}
         {{ form_row(form.dataProvider) }}
         {{ form_row(form.viewType) }}
-        <button class="button m-5">{{ 'planning.search'|trans }}</button>
+        <div>
```
Comment (tuj) [inline templates/planning/index.html.twig:11]: What is the div used for?

**Review by tuj — CHANGES_REQUESTED** [review]

File: templates/planning/index.html.twig (hunk 3 above)
Comment (jeppekroghitk) [reply templates/planning/index.html.twig:11]: To give the button an anchor point in relation to the other form elements, which are wrapped in divs. Now, they are in line. Before, the button were never in line.

**Review by tuj — APPROVED** [review]

**Pull request merged**